intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -86,7 +88,7 @@ def send_trajectory(self, wait_for_result=True):
             return None
         self._check_options()
         self._client.send_trajectory(self.to_msg())
-        return self._client.wait_for_result() if wait_for_result else True
+        return self._client.wait_for_result(timeout) if wait_for_result else None
```

Comment:
This function should return True instead of None if not waiting for the result. This was changed in PR #100.